Workbooks/Workloads/Onboard Redis Insights/Onboard Redis Insights.workbook

```diff
@@ -155,7 +155,7 @@
                 {
                   "type": 1,
                   "content": {
-                    "json": "NGINX Plus is a software load balancer, web server, and content cache built on top of open source NGINX. NGINX Plus has exclusive enterprise‑grade features beyond what's available in the open source offering, including session persistence, configuration via API, and active health checks. Use NGINX Plus instead of your hardware load balancer and get the freedom to innovate without being constrained by infrastructure.\n\nWith Azure Monitor for NGINX Plus, you can manage your NGINX cluster while also gaining visibility into your operating system – Windows or Linux. Make configuration a breeze with templates, automated alerts and customizable actions. Spend less time fighting fires by accurately diagnosing real-time and historical load balancer performance issues, and focus on more proactive measures scale outs.\n<br />\n<br />\n[Watch Video]()<br />\n[Learn More]()<br />\n[Documentation]()<br />"
+                    "json": "Redis is an open source, in-memory data structure store that has rapidly gained popularity from developers. Redis is known for its fast performance and niche as a system that serves as both a store and cache. Redis functions most commonly as a key-value database, but has also gain prominence as a powerful NoSQL database.\n\nAzure Monitor for Redis combines all the important information you need to manage your Redis servers. Azure Monitor for Redis highlights important performance metrics to ensure your Redis server is running at optimal levels. Make configuration a breeze with templates, automated alerts and customizable actions.\n<br />\n<br />\n[Watch Video]()<br />\n[Learn More]()<br />\n[Documentation]()<br />"

```

Comment:
Take out links at the bottom

Reply:
We need content in the pre-requisite tab of MySQL and Redis.

Reply:
sure